util/util_linux.go

```diff
@@ -25,7 +26,14 @@ func mountProc() error {
 	return nil
 }
 
-func Mount(device, directory, fsType, options string) error {
+func Mount(device, directory, fsType string, options_i interface{}) error {
```

Comment:
I used interface purposely to accept both slices and string in this func so but if you think it is excess in this code - I'll get back to string only, no problem